README.md

```diff
@@ -1,2 +1,125 @@
-xblock-google-drive
-=====================
+Google Drive & Calendar XBlock [![Build Status](https://travis-ci.org/edx-solutions/xblock-google-drive.svg?branch=master)](https://travis-ci.org/edx-solutions/xblock-google-drive)
```

Comment:
A minor nit, but it would be great to be in the habit of writing README.rst files instead of README.md files.